### PR TITLE
Added sound to beam animation

### PR DESCRIPTION
**Pull request by MugZitch**

### CODE REVIEW

File: WobblyStealthGame/Assets/Art/Fbx/Beam/Beam.cs [hunk 1]
```diff
@@ -5,12 +5,23 @@
...
         string animationProp = "BeamMeUp";
-        animator.SetBool(animationProp, !animator.GetBool(animationProp));
+        bool playAnimation = !animator.GetBool(animationProp);
+        animator.SetBool(animationProp, playAnimation);
+        if (playAnimation)  
+            audioSource.Play();
+        else
+            audioSource.Stop();
     }
+
```
Comment (ronanhub) [inline WobblyStealthGame/Assets/Art/Fbx/Beam/Beam.cs:26]:
```suggestion
```

File: WobblyStealthGame/Assets/Art/Fbx/Beam/Beam.cs [hunk 2]
```diff
@@ -5,12 +5,23 @@
...
+    public void ToggleBeamAnimation() 
+    {
         string animationProp = "BeamMeUp";
-        animator.SetBool(animationProp, !animator.GetBool(animationProp));
+        bool playAnimation = !animator.GetBool(animationProp);
+        animator.SetBool(animationProp, playAnimation);
+        if (playAnimation)  
+            audioSource.Play();
+        else
+            audioSource.Stop();
```
Comment (ronanhub) [inline WobblyStealthGame/Assets/Art/Fbx/Beam/Beam.cs:24]:
```suggestion
        if (playAnimation)
        {
            audioSource.Play();
        }
        else
        {
            audioSource.Stop();
        }
```

File: WobblyStealthGame/Assets/Art/Fbx/Beam/Beam.cs [hunk 3]
```diff
@@ -5,12 +5,23 @@
 public class Beam : MonoBehaviour
 {
     public Animator animator;
+    public AudioSource audioSource;
```
Comment (ronanhub) [inline WobblyStealthGame/Assets/Art/Fbx/Beam/Beam.cs:8]:
```suggestion
    private Animator animator;
    private AudioSource audioSource;
```

**Review by ronanhub — CHANGES_REQUESTED** [review]
alles werkt maar de code conventions kloppen nog niet

**Review by MatthijsHofele — CHANGES_REQUESTED** [review]
@ronanhub where sound

**Comment by ronanhub** [comment]
> @ronanhub where sound

¯\_(ツ)_/¯

**Comment by ronanhub** [comment]
"added beam sound to prefab"
There sound

**Review by MatthijsHofele — APPROVED** [review]
yes sound good

**Pull request merged**